reference/release-notes/1.28.1.md

```diff
@@ -0,0 +1,124 @@
+---
+orphan: true
+---
+# 1.28.1
+
+These release notes cover new features and changes in Anbox Cloud 1.28.1.
+
+Anbox Cloud 1.28.1 is a patch release. To understand minor and patch releases, see [Release notes](https://documentation.ubuntu.com/anbox-cloud/en/latest/reference/release-notes/release-notes).
+
+Please see [Component versions](https://documentation.ubuntu.com/anbox-cloud/en/latest/reference/component-versions/) for a list of updated components.
+
+## Requirements
+
+See the [Requirements](https://documentation.ubuntu.com/anbox-cloud/en/latest/reference/requirements/) for details on general and deployment specific requirements to run Anbox Cloud.
+
+## New features & improvements
+
+* The LXD charm in Anbox Cloud can be configured to use [Ceph storage](https://ceph.io/en/) in addition to the default local storage options. This helps deployments that require shared storage, improved scalability and better resource utilization.
+* Android security updates for December 2025 (see [Android Security Bulletin - December 2025](https://source.android.com/docs/security/bulletin/2025-12-01)). 
+* The Android WebView has been updated to [143.0.7499.52](https://chromereleases.googleblog.com/2025/12/chrome-for-android-update.html).
+
+## Known issues
+
+See our [open bugs in Launchpad](https://bugs.launchpad.net/anbox-cloud/+bugs?field.searchtext=&orderby=-importance&field.status%3Alist=NEW&field.status%3Alist=CONFIRMED&field.status%3Alist=TRIAGED&field.status%3Alist=INPROGRESS&field.status%3Alist=INCOMPLETE_WITH_RESPONSE&field.status%3Alist=INCOMPLETE_WITHOUT_RESPONSE&assignee_option=any&field.assignee=&field.bug_reporter=&field.bug_commenter=&field.subscriber=&field.structural_subscriber=&field.tag=&field.tags_combinator=ANY&field.has_cve.used=&field.omit_dupes.used=&field.omit_dupes=on&field.affects_me.used=&field.has_patch.used=&field.has_branches.used=&field.has_branches=on&field.has_no_branches.used=&field.has_no_branches=on&field.has_blueprints.used=&field.has_blueprints=on&field.has_no_blueprints.used=&field.has_no_blueprints=on&search=Search).
+
+We assessed the following CVEs from the 2025-12-05 Android Security Bulletin patch for their severity and impact on how Anbox Cloud works, but they are not included with the 1.28.1 release:
+|CVE|Affected components|
+|---|-------------------|
+|[CVE-2025-48551](https://osv.dev/vulnerability/ASB-A-407991863)|AAOS 13|
```

Comment:
Can we give a justification for why we don't include these?

Reply:
The one for AAOS13 is because it's pretty complex to backport from Android 14.
For Android 12, it's a combination of the time it takes, and lower priority for Android 12.